src/main/java/database/controller/DatabaseServiceRestController.java

```diff
@@ -0,0 +1,120 @@
+package database.controller;
+
+import database.entity.BaseEntity;
+import database.service.JsonDatabaseService;
+import io.javalin.config.JavalinConfig;
+import io.javalin.http.BadRequestResponse;
+import io.javalin.http.Context;
+import io.javalin.http.NotFoundResponse;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.javalin.apibuilder.ApiBuilder.*;
+
+public class DatabaseServiceRestController {
+    private static final JsonDatabaseService databaseService = new JsonDatabaseService();
+
+    public void configureRouter(JavalinConfig config) {
+        config.router.apiBuilder(() ->
+                path("/api/v1/database", () -> {
+                    path("/{entityClass}", () -> {
+                        post(this::handleCreateTable);
+                        delete(this::handleDeleteTable);
+                        path("/{entities}", () -> {
```

Comment:
As I see in my tests, `{entities}` path param is not used and can be changed to any value without any effect. Let's change the logic a bit. Try to change endpoints in the following way:

- `/api/v1/database/{entityClass}/table` to create/delete table.
- `/api/v1/database/{entityClass}` to add new record, get/delete all records.
- `/api/v1/database/{entityClass}/{id}` to update/delete record and get record by id.
- `/api/v1/database/{entityClass}/filter` to get records by filters